src/Helpers/Utils.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Godot\AssetLibrary\Helpers;
 
+use PDO;
```

Comment:
The bindValue() method call on line 97 makes use of an explicit parameter type like described here. It's a bit of a doubled safety measure because I cast the value to int anyway.
But I think it's a good habit to use explicit parameter types with bindValue().